thumbnail/thumbnail_magick.go

```diff
@@ -838,6 +838,16 @@ func MakeThumbnailMagick(src io.Reader, dst http.ResponseWriter, params Thumbnai
 
 	//画像出力
 	blob := mw.GetImagesBlob()
+
+	if len(blob) == 0 {
+		err = mw.GetLastError()
+		if err != nil {
+			return errors.New("Magic wand fail: " + err.Error())
```

Comment:
minor: you may do `fmt.Errorf("Magic wand fail: %w", err)` to wrap an error.

Reply:
thanks